exercises/custom-set/custom_set_test.rb

```diff
@@ -3,114 +3,303 @@
 require 'minitest/autorun'
 require_relative 'custom_set'
 
+# rubocop:disable Metrics/LineLength
+
+# Test data version:
+# cdfb4a2
 class CustomSetTest < Minitest::Test
-  def test_equal
-    assert_equal CustomSet.new([1, 3]), CustomSet.new([3, 1])
-    refute_equal CustomSet.new([1, 3]), CustomSet.new([3, 1, 5])
-    refute_equal CustomSet.new([1, 3, 5]), CustomSet.new([3, 1])
-    refute_equal CustomSet.new([1, 3]), CustomSet.new([2, 1])
+  def test_sets_with_no_elements_are_empty
+    # skip
+    set = CustomSet.new []
+    assert_equal CustomSet.new, set
+  end
+
+  def test_sets_with_elements_are_not_empty
+    skip
+    set = CustomSet.new [1]
+    refute_equal CustomSet.new, set
+  end
```

Comment:
I believe this should be that very spec. The metadata for these exercises is:
```json
   "empty": {
     "description": "Returns true if the set contains no elements",
     "cases": [
        "..."
     ]
   },
```
This is not the place to rely on equality. Equality has its own section later on.